core/src/main/java/nu/yona/server/properties/BatchServiceProperties.java

```diff
@@ -8,6 +8,8 @@
 {
 	private String serviceUrl = "http://localhost:8083";
 
+	private String activityAggregationJobCronExpression = "0 0 3 * * *";
```

Comment:
According to [Wikipedia](https://en.wikipedia.org/wiki/Cron) and [this site](http://www.corntab.com/?c=0_0_3_*_*_*), this job now runs at midnight, ever third of every month. That's not what you had in mind, I guess. What about using 0 3 * * * * (i.e. [every day at three in the morning](http://www.corntab.com/?c=0_3_*_*_*_*))?